presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop/apply-site-xml-override.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
`apply-site-xml-override.sh` should be moved, it isn't used by anyone else

if you move this appropriately, you can retain `/docker/presto-product-tests/conf/docker/files/apply-site-xml-override.sh` path in the script